src/crafting.cpp

```diff
@@ -1278,10 +1281,22 @@ void Character::complete_craft( item &craft, const cata::optional<tripoint> &loc
                 if( should_heat || remove_raw ) {
                     comp.set_flag_recursive( flag_COOKED );
                 }
+
+                //when batch crafting, set_components depends on components being merged, so merge any unmerged ones here
```

Comment:
```suggestion
                // when batch crafting, set_components depends on components being merged, so merge any unmerged ones here
```

consistent whitespace with comments